src/views/inventory.js

```diff
@@ -258,7 +258,7 @@ export class inventory {
 
   //Prepack, New Aisle, Old Shelves
   isBin(term) { //unlike shipment page allow for B00* to search all sections within large B00 bin but don't include repacks
-    return /[A-Za-z]\d{2}|[A-Z][1-6][0-6]\d[\d*]|[A-Za-z][0-6]\d[\d*]/.test(term)
+    return /[A-Za-z]\d{2}|[A-Z][1-6][0-6]\d[\d*]|[A-Za-z][0-6]\d[\d*]|\d{1,2}[A-Z]\d{3}/.test(term)
```

Comment:
instead of \d{3} I would do [0-6]\d[\d*] because the bin numbering with our current shelves should stop at 69.

Also we do allow for people to search "B10" is anything in that pre-pack bin, or "B10*" which is anything in that inventory bin.  To allow this functionality in the new bin format you would need to allow the "*" as the regex above does.